lib/xcov/manager.rb

```diff
@@ -219,11 +219,22 @@ def process_xcresults!(xcresult_paths)
             UI.important("Copying .xccovreport to #{path}") 
           end
 
+          if archive_paths.length > 1 && report_paths.length > 1
+            Dir.chdir(output_path) do
+              archive_filenames = archive_paths.map { |path| File.basename(path) }
```

Comment:
```suggestion
              archive_filenames = archive_paths.map { |path| File.basename(path).shellescape }
```

Escape spaces and other characters that can mess up shell commands